src/homedelivery/export.js

```diff
@@ -85,6 +85,15 @@ function getHolidaySuspensions (downloadStream: ReadStream): Promise<Set<string>
       .pipe(csvStream)
   })
 }
+
+function getFullName (zFirstName: string, zLastName: string) {
+  let firstName = zFirstName
+  if (firstName.trim() === '.') {
```

Comment:
I think you should move the .trim() up a line.

Reply:
the reason I didn't do it like that is because the salesforce code doesn't trim the first name. I don't know if we have any names with spaces at the end but if we do they would show up as a difference in the comparator output.

Reply:
Ok cool, we can do this later once we're switched over and not going back.